governance/votes/Executive vote - April 15, 2022.md

```diff
@@ -0,0 +1,93 @@
+* [Auction Price Multiplier (buf)](https://manual.makerdao.com/parameter-index/collateral-auction/param-auction-price-multiplier): 1.00
+* [Local Liquidation Limit (ilk.hole)](https://manual.makerdao.com/parameter-index/collateral-auction/param-local-liquidation-limit): 5 million DAI
+* [Max Auction Duration (tail)](https://manual.makerdao.com/parameter-index/collateral-auction/param-max-auction-duration): 5 days (implies minimum price of 0.98)
+* [Proportional Kick Incentive (chip)](https://manual.makerdao.com/parameter-index/collateral-auction/param-proportional-kick-incentive): Zero DAI
+* [Flat Kick Incentive (tip)](https://manual.makerdao.com/parameter-index/collateral-auction/param-flat-kick-incentive): 500 DAI
+
+For further information, please see the proposal forum thread [here](https://forum.makerdao.com/t/proposed-parameters-for-offboarding-tusd-a/13506).
+
+### CRVV1ETHSTETH-A System Parameters
+
+As per [this](https://vote.makerdao.com/polling/QmeTfgKP#poll-detail) successful governance poll, CRVV1ETHSTETH-A (Curve ETH-stETH pool) parameters will be changed as detailed below if this executive proposal passes.
+* Increase [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 5 million DAI to 50 million DAI.
+* Increase [Target Available Debt](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) from 3 million DAI to 10 million DAI.
+
+For further information, please see the proposal forum thread [here](https://forum.makerdao.com/t/immediate-short-term-parameter-changes-proposal-for-crvv1ethsteth-a-dc-and-gap-increase/14476).
+
+### Growth Core Unit Budget
+
+As per the ratified [GRO-001 DAI Budget](https://mips.makerdao.com/mips/details/MIP40c3SP37#budget-implementation), **474,683 DAI** will be transferred to the [Growth Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4) if this executive proposal passes. This is the second of two lump-sum transfers specified in the budget proposal.
+
+### Fund Ambassador Program Pilot
+
+As per [this](https://vote.makerdao.com/polling/QmPpQ49p#poll-detail) successful governance poll, **25,000 DAI** will be transferred to the Ambassadors Program multi-sig wallet if this executive proposal passes. 
+
+For further information, please see the signal request thread [here](https://forum.makerdao.com/t/signal-request-fund-ambassadors-program-pilot/13199).
+
+### Replace Gelato Keeper Top-Up Contract DAI Stream
+
+As per [this](https://forum.makerdao.com/t/update-to-the-gelato-keeper-network-top-up-contract/14524) forum post, the [Protocol Engineering Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7) have included a replacement for the Gelato Keeper stream in this executive proposal to resolve a bug related to token approvals. This replacement will take place if this executive proposal passes.
+* The existing stream will be revoked.
```

Comment:
What's happening to funds that have been streamed already? Are Gelato working around that at their end?